apps/addon/Dockerfile

```
@@ -16,11 +15,27 @@ COPY apps/addon ./apps/addon
 COPY packages/shared ./packages/shared
 
 RUN pnpm --filter @cataloggy/shared build
-RUN cd apps/addon && pnpm exec tsc -p tsconfig.json
+RUN pnpm --filter @cataloggy/addon build
+
+RUN pnpm prune --prod
+
+FROM node:26-alpine AS runtime
+
+WORKDIR /app
+RUN apk add --no-cache wget
```

Hadolint (2.14.0)

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)